### ISSUE TITLE
Incorrect error when testing even from the test directory.

### ISSUE THREAD

**Issue by saranshdhingra** [GoogleCloudPlatform/php-docs-samples#1429]
While running the bigtable tests, I used the following command:
```
cd bigtable/test
php PATH_TO_TESTING_FOLDER/vendor/bin/phpunit -c ../phpunit.xml.dist ./
```

The error I receive is: `You are not in a test directory`

While checking the `testing/bootstrap.php` file I see that the condition is:

```
if (!file_exists($testDir . '/phpunit.xml.dist')) {
    throw new Exception('You are not in a test directory');
}
```

So, the check is essentially, to check if the current folder has the file `phpunit.xml.dist`. But since, that file doesn't exist in the `test` folder, it may be misleading and can be confusing.

I feel the simplest solution could be to modify the error message :)

**Comment by bshaffer** [comment]
I agree... I think a better error would be "You are not in a sample directory"

**Comment by bshaffer** [comment]
fixed in #1440
